classes/class-wc-connect-help-view.php

```diff
@@ -412,39 +464,9 @@ protected function get_debug_items() {
 				'save_on_toggle' => true,
 			);
 
-			// add connect log tail
-			$log_data = $this->get_debug_log_data();
-			$log_tail_line_count = count( $log_data->tail );
-			if ( $log_tail_line_count < 1 ) {
-				$description = '';
-				$log_tail = __( 'Log is empty', 'woocommerce-services' );
-			} else {
-				$url = add_query_arg(
-					array(
-						'page' => 'wc-status',
-						'tab' => 'logs',
-						'log_file' => $log_data->key
-					),
-					admin_url( 'admin.php' )
-				);
-				$description = sprintf(
-					wp_kses(
-						__( 'Last %d entries <a href="%s">Show full log</a>', 'woocommerce-services' ),
-						array(  'a' => array( 'href' => array() ) ) ),
-					$log_tail_line_count,
-					esc_url( $url )
-				);
-				$log_tail = implode( $log_data->tail, '' );
-			}
-
-			$debug_items[] = (object) array(
-				'key' => 'wcc_debug_log_tail',
-				'title' => __( 'Debug Log', 'woocommerce-services' ),
-				'type' => 'textarea',
-				'description' => $description,
-				'readonly' => true,
-				'value' => $log_tail
-			);
+			$debug_items[] = $this->add_log_view( __( 'Taxes Log', 'woocommerce-services' ), 'taxes' );
+			$debug_items[] = $this->add_log_view( __( 'Shipping Log', 'woocommerce-services' ), 'shipping' );
+			$debug_items[] = $this->add_log_view( __( 'Debug Log', 'woocommerce-services' ) );
```

Comment:
This old title could be revised to disambiguate among the new distinctions — it's not clear that this view is independent/exclusive of a) the other logs, and b) the "Debug" option above. Not sure what to call it though — "Other log"? "Misc log"?

Reply:
Ah, good call. I'll change it to "Other".